programs/gumball-machine/src/state/mod.rs

```diff
@@ -5,7 +5,7 @@ use std::mem::size_of;
 
 #[derive(AnchorSerialize, AnchorDeserialize, Clone, Copy, Zeroable, Pod)]
 #[repr(C)]
-// Current Size: 352 bytes
+// Current Size: 384 bytes
```

Comment:
expose this GUMBALL_MACHINE_HEADER_SIZE as a const if needed

Reply:
I just had this for reference for testing purposes, I can take this out, and move the comment to `gumball-machine-serde.ts` that might be a better place. Its utility was just to facilitate writing client side code

Reply:
It's fine